public/index.html

```diff
@@ -201,11 +196,16 @@ <h2 class="i18n" data-i18n= "ftm-map-of-sites">Donation Sites</h2>
           <p class="i18n" data-i18n="[html]ftm-contact-info">
             Email us at contact@findthemasks.com or join us on <a target="_blank" rel="noreferrer noopener" href="https://github.com/r-pop/findthemasks">our Github</a>.
           </p>
-
           <p class="i18n" data-i18n="[html]ftm-made-with-love">made with &lt;3 in Seattle</p>
           <a href="javascript:;" onclick="window.location='whoweare.html'+window.location.search;" class="end i18n" data-i18n="ftm-who-we-are">who we are</a>
-        </div>
-      </div>
+          <div class="social-media">
+            <a href="http://facebook.com/findthemasks" class="fa fa-facebook social-media-icons" target="_blank"></a>
+            <a href="http://twitter.com/findthemasks" class="fa fa-twitter social-media-icons" target="_blank"></a>
+            <a href="http:#" class="fa fa-linkedin social-media-icons" target="_blank"></a>
```

Comment:
I guess if we don't have linkedin we should just remove it

Reply:
Will do.  Are there any social media icons that you think are missing?

Reply:
Can you replace linkedin with instagram? https://www.instagram.com/findthemasks/